examples/README.md

```diff
@@ -16,38 +16,59 @@ AMOP（Advanced Messages Onchain Protocol）即链上信使协议，旨在为联
 
 ## 单播案例
 
-**单播** 就是节点从监听同一 Topic 的多个订阅者中随机抽取一个订阅者转发消息
+**单播** 指的是节点从监听同一 Topic 的多个订阅者中随机抽取一个订阅者转发消息
 
--   启动 AMOP 服务端：
+-   运行 AMOP 消息发布者：go run examples/amop/unicast_pub/publisher.go [endpoint] [topic]
 
     ```shell
-    go run examples/server/amop_server.go
+    > go run examples/amop/unicast_pub/publisher.go 127.0.0.1:20200 hello
+    
+      2020/08/10 22:43:31 publish message: hello, FISCO BCOS, I am unique broadcast publisher! 0 
+      2020/08/10 22:43:33 PushTopicDataRandom failed, err: sendMessage failed, err: error code 100, remote peer unavailable
+      2020/08/10 22:43:33 publish message: hello, FISCO BCOS, I am unique broadcast publisher! 1 
+      2020/08/10 22:43:35 PushTopicDataRandom failed, err: sendMessage failed, err: error code 100, remote peer unavailable
+      2020/08/10 22:43:35 publish message: hello, FISCO BCOS, I am unique broadcast publisher! 2 
+      2020/08/10 22:43:37 PushTopicDataRandom failed, err: sendMessage failed, err: error code 100, remote peer unavailable
+      2020/08/10 22:43:37 publish message: hello, FISCO BCOS, I am unique broadcast publisher! 3 
+      2020/08/10 22:43:39 publish message: hello, FISCO BCOS, I am unique broadcast publisher! 4
     ```
 
-    控制台输出 "subscribe success" 表示服务启动成功
-
--   修改 config.toml 配置文件中 NodeURL 配置项，从 127.0.0.1:20200 修改为 127.0.0.1:20201，因为 FISCO BCOS 2.5.0 不支持 AMOP 连接同一节点（FISCO BCOS 2.6.0 开始支持连接同一节点）
-
--   运行 AMOP 客户端：
+-   启动 AMOP 消息订阅者：go run examples/amop/sub/subscriber.go [endpoint] [topic]
 
     ```shell
-    go run examples/singleclient/unicast_client.go
+      > go run examples/amop/sub/subscriber.go 127.0.0.1:20201 hello
+      
+      Subscriber success
+      2020/08/10 22:51:57 received: hello, FISCO BCOS, I am unique broadcast publisher! 3
+      2020/08/10 22:51:59 received: hello, FISCO BCOS, I am unique broadcast publisher! 4
     ```
 
-    服务端控制台输出 "hello, unique broadcast!"，客户端输出 "PushTopicDataRandom success" 表示运行成功
+> 注意：控制台输出 *PushTopicDataRandom failed, err: sendMessage failed, err: error code 100, remote peer unavailable* 表示网络中没有对应的 topic 消息订阅者
 
 ## 多播案例
 
-**多播** 就是节点向监听同一个 Topic 的所有订阅者转发消息。只要网络正常，即使没有监听 Topic 的订阅者，消息发布者也会收到消息发送成功的网络回包
+**多播** 指的时节点向监听同一个 Topic 的所有订阅者转发消息。只要网络正常，即使没有监听 Topic 的订阅者，消息发布者也会收到消息发送成功的网络回包
 
--   启动 AMOP 服务端：同单播案例
+-   运行 AMOP 消息发布者：go run examples/amop/multicast_pub/publisher.go [endpoint] [topic]
```

Comment:
Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5979849)